mltrace/cli/cli.py

```diff
@@ -71,8 +185,7 @@ def mltrace():
 @click.option("--url", default="localhost", help="URL of the database.")
 @click.option("--count", default=5, help="Count of recent objects.")
 def recent(
-    url: str,
-    count: int,
+    url: str, count: int,
```

Comment:
I don't think we should set the server's uri if the user doesn't pass in anything. It might be confusing for the user. How do you feel about that?

I think we can make a separate command that the user can access through CLI, a wrapper around `set_server`. That way the user only has to execute that once if using localhost. What do you think?

Reply:
> I don't think we should set the server's uri if the user doesn't pass in anything. It might be confusing for the user.

This does make sense, it is better to constrict the user here.

> I think we can make a separate command that the user can access through CLI, a wrapper around set_server. That way the user only has to execute that once if using localhost.

The problem here is that with a command around `set_server`, once we use the command in CLI, we have a session set with the db server and the session is lost as soon as the CLI command has finished. We would not be able to use the session for other consecutive commands like the `trace` or `history`. This is the sole reason why we need to use the `set_server` command at every step.

PS: If you have a better way to approach this problem I am all up for that. I feel a little skeptical about redoing the `set_server` all the time myself.

Reply:
> The problem here is that with a command around set_server, once we use the command in CLI, we have a session set with the db server and the session is lost as soon as the CLI command has finished.

Good point. I wonder if we can let the server URI be an environment variable actually. What do you think? It might add a bit of complexity / I can make a separate issue for that. But for now, we can keep what you have (except not have a default URI of localhost) and require the user to pass in the URI as a flag.

Reply:
Environment variable sounds like a nice idea! That way things will be easy. A new issue around this might be the way to go! :rocket: